src/storage/src/hummock/shared_buffer/shared_buffer_batch.rs

```diff
@@ -92,6 +92,7 @@ impl SharedBufferBatchInner {
     pub(crate) fn new(
         table_id: TableId,
         epoch: HummockEpoch,
+        gap_epoch: HummockEpoch,
```

Comment:
Instead of passing the `gap_epoch`, we'd better pass the `spill_offset`. And in `LocalHummockStorage` we don't have to maintain the `gap_epoch`.

Reply:
Agree.